scripts/build.sh

```diff
@@ -105,26 +105,16 @@ if [ "$JAVA_HOME" = "" ]; then
     echo "SET JAVA_HOME=$JAVA_HOME"
 fi
 
-# Ensure gcc version is above 4.9.0 for faiss 1.7.4+ compilation
+# Ensure gcc version is above 4.9.0 and is not 8.3.1 for faiss 1.7.4+ compilation
 # https://github.com/opensearch-project/k-NN/issues/975
 GCC_VERSION=`gcc --version | head -n 1 | cut -d ' ' -f3`
 GCC_REQUIRED_VERSION=4.9.0
 COMPARE_VERSION=`echo $GCC_REQUIRED_VERSION $GCC_VERSION | tr ' ' '\n' | sort -V | uniq | head -n 1`
-if [ "$COMPARE_VERSION" != "$GCC_REQUIRED_VERSION" ]; then
-    echo "gcc version on this env is older than $GCC_REQUIRED_VERSION, exit 1"
+if [ "$COMPARE_VERSION" != "$GCC_REQUIRED_VERSION" ] || [ "$GCC_VERSION" = "8.3.1" ]; then
```

Comment:
can we add the link of github issues where we saw that a version of gcc has the problem.

Reply:
It is already there on line 109.

Thanks.